package.json

```diff
@@ -130,6 +132,7 @@
     "vips"
   ],
   "dependencies": {
+    "@types/node": "*",
```

Comment:
Do we need this? My understanding is that it's the responsibility of a consuming TypeScript-based app to define this dependency, rather than doing so in every package that happens to provide type definitions. This approach also avoids the versioning problem you mention.
```sh
$ du -sh node_modules/@types/node
1.9M	node_modules/@types/node
```

Reply:
1.9M worth of typings 👀  Color me surprised.

I honestly don't quite know what the "right" approach is here, or if there even is one:
- A direct dependency will work and have typescript be happy when using the module
- A peer dependency marked as optional might be one option if we want to be explicit about what it depends on
- Dropping it altogether (or rather, moving it to a dev dependency - I think we'll need it for tsd to work) is probably also acceptable, since as you say - people using sharp + typescript is likely to install `@types/node` in their app anyway

Reply:
Let's go with including it in `devDependencies`, which I think is the least intrusive option. We should also add a section to the installation docs about TypeScript support, including the need to install `@types/node` yourself. I can add this later.

I'm also bracing for future PRs that attempt to move `@types/node` to `dependencies` :)

Reply:
> I'm also bracing for future PRs that attempt to move `@types/node` to `dependencies` :)

Haha, it is bound to happen! 👍

Reply:
Moved to dev dependencies and force-pushed.